### PR TITLE
Settings page revamp

### PR DESCRIPTION
**Pull request by dequis**
~~Useless visual noise no one really cares about.~~ This is now about updating the settings doc to the 21st century

### CODE REVIEW

**Comment by dequis** [comment]
Done.

File: documentation/settings/index.markdown [hunk 1]
```diff
@@ -1516,86 +1502,14 @@ From <http://irssi.org/documentation/formats>:
...
-
-## Appendix F is for FAQ.
-{:#a_f}
-
-### Q: Will you relicense this document under YOUR_FAVORITE_LICENSE?
+## Appendix E: Credits and copyright
+{:#a_credits}
 
-A: The Creative Commons Attribution-ShareAlike license best embodies the author's wishes for use and redistribution of his work. It is strictly a license and does not promulgate a political or ideological agenda. YOUR_FAVORITE_LICENSE will be considered if its terms are very similar (if not identical) to the current license, but it will be rejected if it's someone's soapbox.
+Gathered through much effort by Rocco Caputo rcaputo at cpan dot org (aka dngor). Includes original work by Nei, and advice and guidance from irc.freenode.net #irssi.
```
Comment (ailin-nemui) [inline documentation/settings/index.markdown]: double of \/

File: documentation/settings/index.markdown [hunk 2]
```diff
@@ -1516,86 +1502,14 @@ From <http://irssi.org/documentation/formats>:
...
 
-* * *
+We respect the work of others. Parts of this document have been collected from other locations. Wherever possible, we have made every effort to locate and attribute the original authors. Please let us know if we've overlooked you.
 
-### Copyright License.
+We ask the same respect in return.
 
 The original portions of this document are Copyright 2005 by Rocco Caputo rcaputo@cpan.org and Nei (on irc.freenode.net #irssi). Other portions are Copyright by their respective authors or licensors.
```
Comment (ailin-nemui) [inline documentation/settings/index.markdown]: /\

**Comment by ailin-nemui** [comment]
merge it when you think it's ready?

**Pull request merged**